cookbook/security/multiple_guard_authenticators.rst

```diff
@@ -0,0 +1,168 @@
+How to Use Multiple Guard Authenticators
+========================================
+
+Guard authentication component allows you to easily use many different authenticators at a time.
+
+An entry point is a service id (of one of your authenticators) whose start()
+method should be called when an anonymous user hits a page that requires authentication.
```

Comment:
While I know that this is the official description of the entry point, I think we have to change it a bit. An anonymous user can be confused with the `AnonymousToken`. This anonymous token is created by the authentication process (a user is authenticated as being Anonymous).

Afaik, entry points are called when there is no token at all. In such case, the task of an entry point is to start the authentication procedure (e.g. redirect to the login page or return some header that triggers digest authentication).